### ISSUE TITLE
Radar 37

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#439]
What are you working on this week and why?

[last week](https://github.com/gratipay/inside.gratipay.com/issues/429)

**Comment by whit537** [comment]
#### Roadmap ([what?](http://inside.gratipay.com/howto/sweep-the-radar))

###### Short Term

[Embarrassments](https://github.com/gratipay/inside.gratipay.com/issues/368#issuecomment-148587948):

 - can't edit teams ([Team Basics](https://github.com/gratipay/gratipay.com/milestones/Team%20Basics))
 - not using immutable ids (https://github.com/gratipay/gratipay.com/issues/835)
 - widgets are broken (https://github.com/gratipay/inside.gratipay.com/issues/389#issuecomment-153884921)
 - escrow imprecision (https://github.com/gratipay/inside.gratipay.com/issues/389#issuecomment-153929881)
 - Security 9 (https://github.com/gratipay/inside.gratipay.com/issues/389#issuecomment-153930768)
 - no payroll! https://github.com/gratipay/gratipay.com/issues/3433

###### Long Term

 Color code:

 - red—external forces that we're under
 - yellow—administration
 - green—product development
 - orange—marketing
 - blue—capitalization


[![roadmap](https://cloud.githubusercontent.com/assets/134455/11450312/7c0546ac-9569-11e5-957c-4c9898666661.png)](http://interactive.blockdiag.com/?compression=deflate&src=eJyNUsFu2zAMvfsrBO3aXQsURjckXlsE6NAAHnpZdmBkOiUqS4akOEuL_PtE2bGDpWjrk_hE6vG957W26rki2IjXLBMbgFb81rBGfS3vZrOlvBDKauuuHVYXIuDf0Je7Jwr4R2SiVTQOLIuF-FGWH85gM5HcNK22-wZNEPewe380E2sPExukMf-ZOaXHscI2LRjCT8wJdsTZbStee1h-ub0t4ifz5FTO6nOWk_NmeaQ5ZL0lX78JMrX1qHq5sW5h76zWg8kRkHPQYBSKubXPXvbqIj77eZ82jsc726Ez3BTfHR4clSxiXaKSUR9V3QRX0RUK-5V5REc1KQhkDXcNG5w6yPVogtw4RCNPjZDJiURhOkuKzOaEf0D4Oi49XsTzQGe3MZ8jXJFDFVZmwLmlACj_S2YvwK8MiBJdR4qpz0Ko46dUNOzcwsNZc1FwZtx89I-ziX6liAYFfNtDY0pHgIX1OC894bx6rKaETkD5C6ERc_CkvGR8SZ0NfHgwawuuSpTnu7Ksy0vedWEC68edT3_K0qH33_niYUvlEzh8a_zqSqm6Tr4U0FIATS99-Nx9-Ae_HEZp)

**Comment by whit537** [comment]
I'm heading to [COS](https://cos.io/) today and tomorrow. The map I drew to get me there reminds me of our own roadmap: long and winding with a lot of different segments and I can't really keep it in my head all at once.

Should be scenic, though! :-) :blue_car: :deciduous_tree: 

![roadmap](https://cloud.githubusercontent.com/assets/134455/11838990/84b1e280-a3b8-11e5-90e4-5cadb225c0d3.jpg)

**Comment by kzisme** [comment]
@whit537  Are you serious?  I just applied there! :D I had someone who visited there recently recommend I apply. Seems like a cool place!

**Comment by whit537** [comment]
@kzisme Niiiiice! Yeah, it's awesome. Have you heard back from them yet? Definitely use me as a reference! :-()

**Comment by whit537** [comment]
I just accidentally discovered a new emoticon ... :-() horse lips? :horse: :lips:

**Comment by kzisme** [comment]
I haven't heard anything at all - which is sorta expected I guess.  I shall!  What all did you do there?  

The recommendation was from a friend in Openhatch.  

:santa:

**Comment by whit537** [comment]
@kzisme How long has it been? Wanna let me know in private email who you emailed and I can follow up if I know them? I met most of the folks there this trip. I'm going to be helping them with code reviews. They have a *ton* of junior devs because they've had such a strong internship program, but they're light on senior devs. :-)

**Comment by kzisme** [comment]
@whit537 I applied on 12/1 with a resume + cover letter.  Via their online portal - so no email (unless all apps get forwarded to their jobs email address).

It's a bit out of my way geographically - but I read into what they do (Also super into FOSS and the Python community which is what made me really interested)

Could you still reach out on behalf via private email? :)

**Comment by whit537** [comment]
@kzisme Yeah, I'll ping them and see what I can find out ...

**Comment by kzisme** [comment]
Alright I appreciate it @whit537!

**Comment by whit537** [comment]
@kzisme Sent! Now to get you some commits on Gratipay so you have something to show! :dancer: 

/me reopens #253 ...

**Comment by mattbk** [comment]
!m @rohitpaulk. Way to knock some things off the list.

**Comment by rohitpaulk** [comment]
:)

**Comment by whit537** [comment]
> !m @rohitpaulk. Way to knock some things off the list.

Indeed! !m @rohitpaulk 

Perhaps I'm stating the obvious, but I have finally admitted to myself that Gratipay is now back to a side project for me. My wife and I decided on a two-year runway when I first launched Gittip. That ran out just as the Gittip crisis (#319) peaked. To press the metaphor, we were barely off the ground and ended up clipping the cornfield beyond the runway. Then Balanced went under and we hit the Gratipocalypse (#192), which was basically like running out of gas just as we passed the edge of the Grand Canyon. So now we've basically built a dune buggy out of the remains of the plain and are cruising down the canyon. Or something. :-P

Long story short: I'm *really* out of runway, and so we're officially in a new phase now where we're *all* part-time on Gratipay. Kinda burns me up to not get to work on Gratipay full-time, but it seems like we're stabilized, at least, and maybe this is for the best because it gives others a chance to step up and take more ownership (cf. #72)—we're not going to make it otherwise!

Thanks to everyone for hanging in there and stepping up. Let's see what 2016 brings! :-)

**Comment by whit537** [comment]
@kzisme Here's what I heard back from COS: "I'll pass that info on to [our recruiting director] and let him know to contact you should Kevin be a good fit." Here's my email to them:

> I just found out that a friend of mine from the Gratipay project recently applied to COS. His name is Kevin Zita. I believe it was a cold resume submitted through the website on December 1. May I ask who is in charge of recruiting? I'd like to offer myself as a reference for Kevin.

I'll let you know if I hear from them! :-)

**Comment by kzisme** [comment]
Awesome! Thank you again for reaching out!

**Comment by whit537** [comment]
:-)

**Comment by whit537** [comment]
I just ordered 300 single (no carbon-copy) plain business checks from WalmartChecks.com for $27.68. Should be here January 8. We haven't had any checks before and occasionally we need one.

**Comment by whit537** [comment]
I just went through GitHub's organization migration. Not exactly sure what I did, it's in organization settings. Not finding a blog post about it, but I did find this!

> #### Weekly radar meetings (standup)
>
Each Monday, we create an issue that we use as our standup meeting. We add our availability for the week and the main items we will be focusing on. We get together for a 30-minute video chat session to collaborate on the week's activities and to spend some face-to-face time together. People add updates to their activities throughout the week in the radar issue. This helps us stay connected as a distributed team.

https://github.com/blog#weekly-radar-meetings-standup

:dancer:

**Comment by whit537** [comment]
Golly. I totally forgot to make a new Radar ticket, didn't I! :flushed:

**Comment by whit537** [comment]
#### Email ([what?](http://inside.gratipay.com/howto/sweep-the-radar))

![screen shot 2015-12-22 at 6 29 14 pm](https://cloud.githubusercontent.com/assets/134455/11968402/f1bedcb8-a8d9-11e5-9e84-85e4e480c8bf.png)

**Comment by whit537** [comment]
> We haven't had any checks before and occasionally we need one.

https://github.com/gratipay/inside.gratipay.com/issues/319#issuecomment-166757830 ;-)
